R/data_rana.R

```diff
@@ -0,0 +1,27 @@
+#'The Twins’ Magical Run Will Likely Go Through The Yankees. (Because Of Course.)
+#' 
+#' The raw data behind the story 
+#' "The Twins’ Magical Run Will Likely Go Through The Yankees. (Because Of Course.)"
+#' \url{https://fivethirtyeight.com/features/the-yankees-always-have-the-twins-number-will-this-postseason-be-different/}
+#' 
+#'   @format A data frame contains 98,796 rows representing Quasi-Win Shares (QWS), prorated to 162 team games, for every player from 1901 as of Sept. 27, 2019 and,  and 16 variables:
+#' \describe{
+#'   \item{name_common}{Player name}
+#'   \item{age}{Player's age on June 30 of season}
+#'   \item{player_id}{Player's Baseball-Reference.com ID}
```

Comment:
Could you make this a clickable link using `\url{}`? Be sure to test in resulting help file.